frame/contracts/src/benchmarking/mod.rs

```diff
@@ -293,18 +286,16 @@ benchmarks! {
 		let addr = Contracts::<T>::contract_address(&caller, &hash, &input, &salt);
 	}: _(origin, value, Weight::MAX, None, code, input, salt)
 	verify {
-		// the contract itself does not trigger any reserves
-		let deposit = T::Currency::reserved_balance(&addr);
+		let deposit_account = Contract::<T>::address_info(&addr)?.deposit_account().clone();
+		let deposit = T::Currency::free_balance(&deposit_account);
```

Comment:
Since the contract address (and hence the deposit address) can be somewhat choosen by the deployer, in *theory* it should be possible to find a specific address (or an already "existing") address? But I'm not sure how practical this is.

Reply:
Yes. The inputs to the hash function can be chosen to a degree. But it is one of the fundamental guarantees of a cryptographic hash function that you can't find the inputs (without brute force) to produce a given output (pre-image resistance).

Reply:
Sure, I don't think this is practical to brute.